src/app/shared/models/bet-status.model.ts

```diff
@@ -0,0 +1,5 @@
+export enum BetStatus {
+  'Win' = 'Win',
+  'Lose' = 'Lose',
+  'In Progress' = 'In Progress',
```

Comment:
`InProgress = 'IN_PROGRESS'` - this is default notation for enums, so in code you can write `BetStatus.InProgress` and not `BetStatus['In Progress']`

Reply:
Changed, thx 
But in my implementation i need version like `InProgress = 'In Progress',`